test/extended/testdata/builds/volumes/docker-deploymentconfig.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
non-blocking nit/note - moving forward we should prefer using `Deployment` in our tests. However, it looks like the test utilizes an imagestream + utility methods in our e2e framework, so DeploymentConfig works better from an ease of use perspective.